marconi-chain-index/src/Marconi/ChainIndex/Indexers/Utxo.hs

```diff
@@ -300,7 +302,8 @@ instance FromJSON UtxoRow where
             (Just s', Just bh', Just txId') -> Just $ SpentInfo (ChainPointRow s' bh') txId'
             _error -> fail "Inconsistent spent info"
      in UtxoRow
-          <$> v .: "utxo"
+          <$> v
+            .: "utxo"
```

Comment:
weird linebreak... was it introduced by fourmolu?

Reply:
I got the same thing, in VIM but only when using formoulu with HLS. Using `fix-fourmolu` from the CLI didn't introduce this weird linebreak.